### PR TITLE
Ageist homework

### PR DESCRIPTION
**Pull request by sarahmaria398**
drinking age

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -18,9 +18,10 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```
Comment (nigelr) [inline Gemfile.lock:21]: Is this something needed for Windows specifically?

File: models/human.rb [hunk 2]
```diff
@@ -17,13 +17,34 @@ def legal_name
...
+#    end
+
+def to_drink
+  @to_drink
+end
 
-  def job_title
-    @job_title
-  end
+def to_drink
```
Comment (nigelr) [inline models/human.rb]: you have to_drink defined twice

File: models/human.rb [hunk 3]
```diff
@@ -17,13 +17,34 @@ def legal_name
...
+  if @age >= '18'
+    to_drink = true
+  else
+    to_drink = false
+end
```
Comment (nigelr) [inline models/human.rb]: make sure `end` are aligned with the above `if`

File: models/human.rb [hunk 4]
```diff
@@ -17,13 +17,34 @@ def legal_name
...
+
+def to_drink
+  @to_drink
+end
 
-  def job_title
-    @job_title
-  end
+def to_drink
+  if @age >= '18'
```
Comment (nigelr) [inline models/human.rb]: you are referring to `age` as a string not a number

Comment (nigelr) [reply models/human.rb]: You did not need the `if` here

File: specs/models/human_spec.rb [hunk 5]
```diff
@@ -39,9 +39,42 @@
...
+  it 'it sets the age for human instance' do
+    human = Human.new('')
+    expect(human).to respond_to(:age)
+  end
+end
+
+describe '.age' do
+  it 'it gets the age for the human instance' do
+    human = Human.new('')
+    human.age = '34'
```
Comment (nigelr) [inline specs/models/human_spec.rb]: `age` should be a number not a string

File: models/human.rb (hunk 2 above)
Comment (nigelr) [reply models/human.rb]: `to_drink` returns a boolean so it is better to name it `to_drink?`

**Review by nigelr — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 1 above)
Comment (sarahmaria398) [reply Gemfile.lock:21]: no idea...

File: models/human.rb (hunk 4 above)
Comment (sarahmaria398) [reply models/human.rb]: I remove 'if' and it gets annoyed about to_drink = true

File: models/human.rb (hunk 2 above)
Comment (sarahmaria398) [reply models/human.rb]: I rename all to to_drink? and it gets annoyed with to_drink = true (unexpected = )

**Pull request closed without merging**